src/ray/gcs/gcs_server/test/gcs_actor_scheduler_test.cc

```diff
@@ -376,6 +385,37 @@ TEST_F(GcsActorSchedulerTest, TestSpillback) {
   ASSERT_EQ(actor->GetWorkerID(), worker_id);
 }
 
+TEST_F(GcsActorSchedulerTest, TestReschedule) {
```

Comment:
should test 2 cases: 1) actor is already tied to a leased worker; 2) not tied.

Reply:
fixed